zetaclient/evm/evm_signer.go

```diff
@@ -32,37 +37,37 @@ const (
 	ZetaBlockTime             = 6500 * time.Millisecond
 )
 
-type EVMSigner struct {
-	client                      EVMRPCClient
+type Signer struct {
```

Comment:
maybe leave `EVMSigner` untouched is good to indicates its purpose?

Reply:
Similar to the other comment. When we do make lint, it will return an error with redundant naming since it is already in the evm package.